core/server/data/migrations/versions/3.22/04-populate-settings-groups-and-flags.js

```diff
@@ -0,0 +1,104 @@
+const logging = require('../../../../../shared/logging');
+
+const typeGroupMapping = [{
+    keys: [
+        'members_public_key',
+        'members_private_key',
+        'members_email_auth_secret'
+    ],
+    from: 'members',
+    to: 'core'
+}, {
+    keys: [
+        'members_private_key',
+        'members_email_auth_secret',
```

Comment:
These two fields pop up in both `site` and `core` groups. I believe they should have been left in `core` group :thinking:

Reply:
It also breaks the "down" migration - leaves the field in "blog" type.

Reply:
🤦‍♂️ nice 🎣 thanks!